ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -202,25 +216,45 @@ open class TwoLineTitleView: UIView {
     // MARK: Setup
 
     /// Sets the relevant strings and button styles for the title and subtitle.
+    /// This is a legacy API that maintains Objective C compatibility with older versions of fluentui-apple.
     ///
     /// - Parameters:
     ///   - title: A title string.
     ///   - subtitle: An optional subtitle string. If nil, title will take up entire frame.
     ///   - interactivePart: Determines which line, if any, of the view will have interactive button behavior.
     ///   - accessoryType: Determines which accessory will be shown with the `interactivePart` of the view, if any. Ignored if `interactivePart` is `.none`.
     @objc open func setup(title: String, subtitle: String? = nil, interactivePart: InteractivePart = .none, accessoryType: AccessoryType = .none) {
+        setup(title: title, subtitle: subtitle, alignment: .center, interactivePart: interactivePart, animatesTextColor: true, accessoryType: accessoryType)
+    }
+
+    /// Sets the relevant strings and button styles for the title and subtitle.
+    ///
+    /// - Parameters:
+    ///   - title: A title string.
+    ///   - subtitle: An optional subtitle string. If nil, title will take up entire frame.
+    ///   - alignment: How to align the title and subtitle. Ignored if `subtitle` is nil.
+    ///   - interactivePart: Determines which line, if any, of the view will have interactive button behavior.
+    ///   - animatesTextColor: If true, the text color will flash when pressed. Ignored if `interactivePart` is `.none`.
```

Comment:
Does design specify that this is the default behavior?

Reply:
Also if there's an animated component, could you take a gif screenshot to show the behavior before and after?

Reply:
As far as the new `alignment` and `animatesTextColor` parameters go, the default values are their current behaviors.